cluster-api/Makefile

```diff
@@ -0,0 +1,55 @@
+TARGET_OS_ARCH:=$(shell go env GOOS)_$(shell go env GOARCH)
+
+TFSUBDIRS:=	$(foreach DIR,$(shell find providers -maxdepth 1 -mindepth 1 -type d),$(subst providers/,,$(DIR)))
```

Comment:
The `TF-` prefix refers to terraform so let's drop it. Perhaps `PROVSUBDIRS` (provider subdirectories)?

Reply:
Done!